commands/pipelines/info.js

```diff
@@ -22,32 +32,49 @@ module.exports = {
     {name: 'pipeline', description: 'pipeline to show', optional: false}
   ],
   flags: [
-    {name: 'json', description: 'output in json format'}
+    {name: 'json', description: 'output in json format'},
+    {name: 'with-owners', description: 'shows owner of every app', hidden: true}
   ],
   run: cli.command(co.wrap(function* (context, heroku) {
     const pipeline = yield disambiguate(heroku, context.args.pipeline)
 
-    const apps = yield listPipelineApps(heroku, pipeline.id)
+    let apps = yield listPipelineApps(heroku, pipeline.id)
+    const reviewApps = sortBy(apps.filter(app => app.coupling.stage === 'review'), ['name'])
+    const stagingApps = sortBy(apps.filter(app => app.coupling.stage === 'staging'), ['name'])
+    const productionApps = sortBy(apps.filter(app => app.coupling.stage === 'production'), ['name'])
```

Comment:
`developmentApps` is missing here. I ran into this today by chance, debugging customer issues and couldn't figure out why their review app parent wasn't in the pipeline. It was in the development lane...

Reply:
@joshwlewis oh!, thought we wanted to hide them on purpose. Will add